masterlist.yaml

```
@@ -24908,3 +24908,11 @@ plugins:
   - name: 'MarkekrausExitCombatPower.esl'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/58651/' ]
     msg: [ *obsolete ]
+
+  - name: 'TheScarlett.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/2434/' ]
```

The location would benefit from a name:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/2434/'
        name: 'The Scarlett - A Buildable Ship'
```